### PR TITLE
Clarify that candidate type field is not related to whether a candidate is mDNS or not

### PR DESCRIPTION
**Pull request by youennf**
Fixes https://github.com/rtcweb-wg/mdns-ice-candidates/issues/120

### CODE REVIEW

File: draft-ietf-rtcweb-mdns-ice-candidates.md [hunk 1]
```diff
@@ -334,8 +334,10 @@ defeating the value of the mDNS wrapping.
 
 To prevent disclosure of the host IP address to a TURN server, the ICE
 agent MUST NOT form candidate pairs between its own relay candidates
-and remote mDNS candidates. Note that the converse is not an issue; the
-ICE agent MAY form candidate pairs between its own mDNS candidates and
+and remote mDNS candidates. Note that filtering out on candidate type
+field is not enough as remote mDNS candidates are identified from
```
Comment (tQsW) [inline draft-ietf-rtcweb-mdns-ice-candidates.md:338]: I tweaked the wording a bit for your reference
```suggestion
This restriction must not be limited to remote mDNS candidates of a specific candidate type, typically host, as mDNS candidates are identified from their connection-address fields.

File: draft-ietf-rtcweb-mdns-ice-candidates.md [hunk 2]
```diff
@@ -334,8 +334,10 @@ defeating the value of the mDNS wrapping.
...
+their connection-address field. Note also that the converse is not an issue:
```
Comment (tQsW) [inline draft-ietf-rtcweb-mdns-ice-candidates.md:339]: nit
```suggestion
their connection-address field. Note also that the converse is not an issue;
```

**Review by tQsW — APPROVED** [review]

File: draft-ietf-rtcweb-mdns-ice-candidates.md [hunk 3]
```diff
@@ -334,8 +334,10 @@ defeating the value of the mDNS wrapping.
 
 To prevent disclosure of the host IP address to a TURN server, the ICE
 agent MUST NOT form candidate pairs between its own relay candidates
-and remote mDNS candidates. Note that the converse is not an issue; the
-ICE agent MAY form candidate pairs between its own mDNS candidates and
+and remote mDNS candidates. Note that filtering out on candidate type
```
Comment (youennf) [inline draft-ietf-rtcweb-mdns-ice-candidates.md:337]:
```suggestion
and remote mDNS candidates.
```

File: draft-ietf-rtcweb-mdns-ice-candidates.md [hunk 4]
```diff
@@ -334,8 +334,10 @@ defeating the value of the mDNS wrapping.
...
+This restriction must not be limited to remote mDNS candidates of a specific candidate type, typically host, as mDNS candidates are identified from their connection-address fields.
```
Comment (youennf) [inline draft-ietf-rtcweb-mdns-ice-candidates.md:338]:
```suggestion
This restriction must not be limited to remote mDNS candidates of a specific candidate type,
typically host, as mDNS candidates are identified from their connection-address fields.
```

**Pull request merged**